gauges/replication.go

```diff
@@ -51,30 +51,33 @@ func (g *Gauges) StreamingWALs() prometheus.Gauge {
 	)
 }
 
-// ReplicationLag returns a prometheus gauge for the database replication
-// lag in milliseconds
-func (g *Gauges) ReplicationLag() prometheus.Gauge {
+// ReplicationDelay returns a prometheus gauge for the database replication
+// lag in seconds
+func (g *Gauges) ReplicationDelay() prometheus.Gauge {
+	return g.new(
+		prometheus.GaugeOpts{
+			Name:        "postgresql_replication_delay",
+			Help:        "Dabatase replication delay in seconds",
+			ConstLabels: g.labels,
+		},
+		"SELECT coalesce(extract(epoch from now() - pg_last_xact_replay_timestamp()), 0) AS replication_delay",
+	)
+}
+
+// ReplicationDelayInBytes returns a prometheus gauge for the database replication
+// lag in bytes
+func (g *Gauges) ReplicationDelayInBytes() prometheus.Gauge {
 	version := postgres.Version(g.version())
 
 	return g.new(
 		prometheus.GaugeOpts{
-			Name:        "postgresql_replication_lag",
-			Help:        "Dabatase replication lag",
+			Name:        "postgresql_replication_delay_in_bytes",
```

Comment:
`postgresql_replication_delay_bytes` to follow the same convention of other proposed changes